lib/beaker/dsl/helpers.rb

```diff
@@ -23,14 +24,12 @@ module DSL
     #
     #
     module Helpers
-      include Beaker::DSL::Helpers::FacterHelpers
       include Beaker::DSL::Helpers::HostHelpers
-      include Beaker::DSL::Helpers::PuppetHelpers
       include Beaker::DSL::Helpers::TestHelpers
-      include Beaker::DSL::Helpers::TKHelpers
```

Comment:
@er0ck yeah, beaker-facter should be going away. It (and beaker-hiera) were good learning exercises, but they don't appear to be enough code at this point to justify existing separately from beaker-puppet. The same is true of tk-helpers. Let me know if you think that's wrong, and if so, we can discuss this further.